### PR TITLE
create tabs at image builder table with federation model

### PR DESCRIPTION
**Pull request by mgold1234**
this pr depends on Ana's pr that implements the federation approach which gives us the ability to expose components in edge frontend repository and use that components at this repository 
related to https://issues.redhat.com/browse/THEEDGE-3072

### CODE REVIEW

File: src/Components/ImagesTable/ImagesTable.js [hunk 1]
```diff
@@ -215,111 +208,90 @@ const ImagesTable = () => {
...
-                  return (
-                    <Tbody key={id} isExpanded={isExpanded(compose)}>
-                      <Tr className="no-bottom-border">
-                        <Td
-                          expand={{
-                            rowIndex,
-                            isExpanded: isExpanded(compose),
-                            onToggle: () =>
-                              handleToggle(compose, !isExpanded(compose)),
-                          }}
```
Comment (croissanne) [inline src/Components/ImagesTable/ImagesTable.js]: Maybe just drop the first 2 commits? Seems like they're negated here anyway.

File: src/Components/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -94,92 +100,116 @@ export const LandingPage = () => {
...
-                iconPosition="right"
-                variant="link"
+      <Tabs
+        className="pf-u-ml-md"
+        activeKey={activeTabKey}
+        onSelect={handleTabClick}
+      >
+        <Tab
+          eventKey={0}
+          title={<TabTitleText>Traditional (RPM - DNF )</TabTitleText>}
```
Comment (croissanne) [inline src/Components/LandingPage/LandingPage.js]: Space between DNF and the closing bracket `)`.

**Review by croissanne — CHANGES_REQUESTED** [review]
Thank you! Some nitpicks, mostly code cleanup:

- follow https://www.osbuild.org/guides/developer-guide/workflow.html#commits for the format of the commit messages.
- please squash these commits, the first 2 commits don't contribute much in terms of git history, as they're immediately overwritten by the last commit.
- split out the webpack config changes out to a separate commit with some details describe what was changed and what the required format of the added variables is.

File: src/Components/ImagesTable/ImagesTable.js (hunk 1 above)
Comment (mgold1234) [reply src/Components/ImagesTable/ImagesTable.js]: good advice, will do it

**Comment by croissanne** [comment]
Closed in favour of #1151

**Pull request closed without merging**